### ISSUE TITLE
Allow inline @var annotation without variable name above return statement

### ISSUE THREAD

**Issue by ADmad** [slevomat/coding-standard#851]
Static analysers like phpstan and psalm allow using inline `@var` annotation without variable name above `return` statements. For e.g.

```php
/** @var string */
return $var;
```

Currently for such usage the sniff `SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration.InvalidFormat` throws error like `Invalid inline documentation comment format "@var string", expected "@var type $variable"`.

**Comment by kukulich** [comment]
Implemented in https://github.com/slevomat/coding-standard/commit/01f73736ce84dbf9e1ca1b071a899b702e5eb7de
